integrationTests/multiShard/block/executingMiniblocks_test.go

```diff
@@ -118,7 +127,7 @@ func TestShouldProcessBlocksInMultiShardArchitecture(t *testing.T) {
 			err := n.blkProcessor.ProcessBlock(
 				n.blkc,
 				n.headers[0],
-				block.Body(n.miniblocks),
+				getBlockBody(testMarshalizer, testHasher, n.headers[0], n.miniblocks),
```

Comment:
why is this needed ? n.miniblocks should only have the miniblocks to this block.

Reply:
Yes, but in the mess of this test file, n.miniblocks returned the miniblocks in the same order they were retrieved, leading to sync errors. getBlockBody actually get the miniblocks in the same order as described in the header. After refactoring is done, getBlockBody will be discarded, having the much more better func TestProcessorNode.SyncBlock() that does the same thing in a more elegant manner.